src/test/java/com/ericsson/ei/queryservice/test/QueryServiceRESTAPITest.java

```diff
@@ -121,20 +115,19 @@ public void getQueryAggregatedObjectTest() throws Exception {
     public void getQueryMissedNotificationsTest() throws Exception {
         ArrayList<String> response = new ArrayList<String>();
         response.add(missedNotification);
-        String expectedOutputWithSquareBrackets = FileUtils.readFileToString(new File(missedNotificationOutputPath),
+        String expectedOutputWithSquareBrackets = FileUtils.readFileToString(new File(failedNotificationOutputPath),
                 "UTF-8");
         String expectedOutput_string = (expectedOutputWithSquareBrackets.substring(1,
                 expectedOutputWithSquareBrackets.length() - 1));
         JsonNode expectedOutput = mapper.readTree(expectedOutput_string);
         LOGGER.info("The expected output is : " + expectedOutput.toString());
 
-        Mockito.when(missedNotificationController.getQueryMissedNotifications(Mockito.anyString()))
+        Mockito.when(failedNotificationController.getFailedNotifications(Mockito.anyString()))
                 .thenReturn(new ResponseEntity(response.get(0), HttpStatus.OK));
 
-        RequestBuilder requestBuilder = MockMvcRequestBuilders.get("/queryMissedNotifications?")
-                                                              .accept(MediaType.APPLICATION_JSON)
-                                                              .param("SubscriptionName",
-                                                                      "Subscription_1");
+        RequestBuilder requestBuilder = MockMvcRequestBuilders.get("/failed-notifications?")
+                .accept(MediaType.APPLICATION_JSON).param("SubscriptionName", "Subscription_1");
```

Comment:
This mocks the wrong parameter, 'subscriptionName'. Also check that whatever mocked request is done with the correct  parameter as well.

Reply:
I changed it with "subscriptionName"